### DISCUSSION TITLE
Unable to add Renovate to Repo - "Cannot subscribe to a retired plan"

### DISCUSSION THREAD

**Discussion by kevindice** [renovatebot/renovate#24629] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us what version of Renovate you run.

n/a

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

When I try to add Renovate to my GitHub organization, I'm unable.

Repro:

- Go here: https://github.com/marketplace/renovate
- Press the "Install it for Free" button
- Get this error:

<img width="1512" alt="Screenshot 2023-09-23 at 10 03 01 PM" src="https://github.com/renovatebot/renovate/assets/5779569/30527426-1540-445c-a52d-4fd16c8c5600">

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
It seems to be a GitHub UI bug. Try this:

<img width="1050" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/1ad86f99-21fe-4ff9-9c10-bcadd17851af">

And FYI Renovate doesn't need a plan at all - you can also go directly to https://github.com/apps/renovate
